layouts/partials/footer.html

```diff
@@ -13,7 +13,7 @@
               <span>Found a bug? Looking for something new?</span>
             </div>
             <div class="col-lg d-none d-md-block pl-0">
-              <button class="button footer-button"><a href="https://docs.nginx.com/feedback/" alt="Select to open the submit feedback form">Let Us Know</a></button>
+              <button class="button footer-button"><a href="/feedback/" alt="Select to open the submit feedback form">Let Us Know</a></button>
```

Comment:
With https://github.com/nginxinc/docs/pull/985/files, it's no longer a "feedback form", so maybe:

```suggestion
              <button class="button footer-button"><a href="/feedback/" alt="Select if you have feedback">Let Us Know</a></button>
```